spec/fake_braintree/subscription_spec.rb

```diff
@@ -88,6 +88,12 @@
     end
   end
 
+  it "sets the creation time" do
+    subscription = create_subscription.subscription
+
+    creation_time = Time.parse(subscription.created_at)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.